CHANGELOG.md

```diff
@@ -51,6 +51,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * `jj log` can now show cryptographic commit signatures. The output can be
   controlled by the `ui.show-cryptographic-signatures=true` config knob.
 
+* This release takes the first steps make target revision a required argument of
+  `bookmark create`, `bookmark move` and `bookmark set`. Those commands will
+  display a warning if the user does not specify target revision  explicitly. In
+  the near future those commands will fail if target revision is not specified.
+
```

Comment:
This should be moved to "[Unreleased]" above. I think "Deprecations" is the right place for it.

Reply:
Done. I also tweaked the wording.